src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -560,16 +560,17 @@ void ResolveContext::DispatchSuccess()
     }
 }
 
-bool ResolveContext::TryReportingResultsForInterfaceIndex(uint32_t interfaceIndex)
+bool ResolveContext::TryReportingResultsForInterfaceIndex(uint32_t interfaceIndex, std::string domainName)
```

Comment:
Why is name passed by value?  I'd think it should be const reference.

And really, the argument should be a const reference to the key type...

Reply:
done.